src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/caseupdate/controllers/UpdateCaseController.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.reform.bulkscanccdeventhandler.caseupdate.controllers;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.caseupdate.model.in.CaseUpdate;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.caseupdate.model.out.SuccessfulUpdateResponse;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.caseupdate.services.CaseUpdater;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.common.auth.AuthService;
+
+import javax.validation.Valid;
+
+@RestController
+public class UpdateCaseController {
+
+    private final AuthService authService;
+    private final CaseUpdater caseUpdater;
+
+    // region constructor
+    public UpdateCaseController(
+        AuthService authService,
+        CaseUpdater caseUpdater
+    ) {
+        this.authService = authService;
+        this.caseUpdater = caseUpdater;
+    }
+    // endregion
+
+    @PostMapping("/update-case")
+    public SuccessfulUpdateResponse updateCase(
+        @RequestHeader(name = "ServiceAuthorization", required = false) String serviceAuthHeader,
```

Comment:
according to the tech spec, this is true. please check

Reply:
it's used like that so that we can throw a custom exception which is translated to 401. otherwise this method will not even be called an Spring will generate 400 response automatically.

Reply:
(same thing is done in the other 2 endpoints)